app/routes/public/sessions.js

```diff
@@ -87,6 +87,8 @@ export default class SessionsRoute extends Route {
       });
     }
 
+    const isFiltering = (params.room === null && params.track === null && params.data === null) ? false : true;
```

Comment:
First, shouldn't it be params.date and not params.data?

```suggestion
    const isFiltering = params.room || params.track || params.date;
```

And please check if this will work

Reply:
Yeah, I tried as you proposed at first, but if any one factor is 0(zero), isFiltering is evaluated to false.
So to get correct filtering info, I think my code should be used, but params.data is my mistake.
Is it okay to change only params.data to params.date?

Reply:
Neither room, track or date can be 0, so my recommendation will make the code very small

Reply:
If you think it is better, let's go on that way.